landscape.yml

```diff
@@ -3508,11 +3512,17 @@ landscape:
             crunchbase: https://www.crunchbase.com/organization/netflix
           - item:
             name: Xline
+            homepage_url: https://www.xline.cloud
```

Comment:
```suggestion
```

Reply:
Not quite sure what should I change?

Reply:
Ohh sorry I see - this is a duplicated entry.